index.html

```diff
@@ -8,50 +8,266 @@
+                    <span class="skils">Ruby on rails</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/gym-fit-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Tonic</h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="card">
+        <div class="card-items">
+            <img src="./assets/art-mobile-bg.png" alt="" />
+            <div class="card-continent">
+                <div>
+                    <h1 class="card-header">Multi-Post Stories
+                    </h1>
+                </div>
+                <div class="cards-elemnts">
+                    <span class="span-weight-500">CANBOY</span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">Back End Dev </span>
+                    <img src="./assets/counter.png" alt="" />
+                    <span class="span-weight-400">2018</span>
+                </div>
+                <p class="card-text">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <div>
+                    <span class="skils">HTML</span>
+                    <span class="skils">CSS</span>
+                    <span class="skils">JavaScript</span>
+                </div>
+                <div class="button-container">
+                    <p class="button">See Project</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about ">
+        <div class="card-items">
+            <div class="card-continent">
+                <h1>About <br/> Myself</h1>
```

Comment:
- [ ] There should be only one `<h1>` tag on a page for semantic and SEO reasons. Kindly replace `<h1>` tag other than the one used for your name in the first section with `<hx>` tags (h2. .... h6).